### DISCUSSION TITLE
multiple tags on same commit succeeded only with first tag

### DISCUSSION THREAD

**Discussion by ashraffouda** [goreleaser/goreleaser#3732] [Q&A] [answered]
I have this issue if I have a tag `v1.6.0-rc1` on commit `b5a4ed1e1edc8ce5807a59972990ce56d4dd8d08` it works well but if I did another tag v1.6.0 on same commit `b5a4ed1e1edc8ce5807a59972990ce56d4dd8d08` again this will try to release `v1.6.0-rc1` again  not v1.6.0 any clues how to fix this ?

**Comment by caarlos0** [comment, marked as answer]
2 options:
1. https://goreleaser.com/customization/git
2. https://goreleaser.com/cookbooks/set-a-custom-git-tag

**Comment by ashraffouda** [comment]
thank u so much, your comment helped a lot  I was able to trigger the workflow with the right tag, but I had to create annotated tags and make goreleaser to sort with 
```
git:
  tag_sort: taggerdate
```
